doc/LABELS.md

```diff
@@ -1,38 +1,39 @@
-# Labels used at Public Lab
-At Public Lab, we use many different labels and tags to keep up with issues and pull requests.
++# Labels used at Public Lab :label:
++At Public Lab, we use labels to categorize and keep up with issues and pull requests.
```

Comment:
@tejaswari7 Can you remove the `+` signs in front of the new lines you added? Thanks!